scripts/visualize.py

```
@@ -0,0 +1,692 @@
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import threading
+from functools import lru_cache
+from http.server import BaseHTTPRequestHandler, ThreadingHTTPServer
+from pathlib import Path
+from typing import Any, Dict, List
```

Ruff (0.14.4)

`typing.Dict` is deprecated, use `dict` instead

(UP035)